lib/model/graphParser.js

```diff
@@ -1,5 +1,6 @@
 var Graph = require('./graph');
 var Connection = require('./connection');
+/*global Node:true*/
```

Comment:
This is a hint to eslint that we are aware of the fact that Node is defined somewhere else. This line should be removed when the name change from Node to Vertex is complete.